exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/using_a_script_languages_container.ipynb

```diff
@@ -0,0 +1,704 @@
+{
```

Comment:
<blockquote>I think the title is incorrect. The notebook is mostly about building a customised SLC, not using it.</blockquote><p>I changed the title to "Using the Script-Languages Container Tool"</p><blockquote>The notebook is quite big. We usually split such notebooks into several parts. For instance you can divide this one into building and testing.</blockquote><p>Split up the notebook in 4 parts.</p><blockquote>It would be good to have a short overview of the whole process. A couple of sentences like "We are going to build a customised language container with added package xgboost ..." would help understanding a lot. Otherwise it is easy to lose the sight of what's happening in the plain list of steps.</blockquote><p>I added more explanation.</p><blockquote>Regarding the style, it's better to be consistent with the use of "we"/"you". My personal preference is "we", i.e. "We are going to do this an that". "Let's do this" etc.</blockquote><p>Changed all the comments, and use only the second person.</p>